ca/ca.go

```diff
@@ -529,6 +547,8 @@ func (ca *CertificateAuthorityImpl) issueCertificateOrPrecertificate(ctx context
 		},
 		Serial:     serialBigInt,
 		Extensions: extensions,
+		NotBefore:  validity.NotBefore,
+		NotAfter:   validity.NotAfter,
```

Comment:
If you remove these two added lines, you can see that everything still builds and runs but the new test case fails, thus showing that adding these two lines (and everything they depend on) fix the problem.